### ISSUE TITLE
Flex gap isn't reported

### ISSUE THREAD

**Issue by Razunter** [ismay/stylelint-no-unsupported-browser-features#273]
`gap` property can be used both in grids and flex, but browser support for these cases is different. Currently, plugin doesn't detect `gap` usage with `display: flex`

There should be at least a simple check for both these properties in one rule:
```
      display: flex;
      gap: 20px;
```

[MDN](https://developer.mozilla.org/en-US/docs/web/CSS/gap#browser_compatibility)

Testing with
```
  "browserslist": [
    "last 3 versions",
    "> 0.2%",
    "safari >= 10",
    "ios_saf >= 10",
    "not IE < 12",
    "not ie_mob < 12",
    "not op_mini all",
    "not dead"
  ],
```

**Comment by ismay** [comment]
See this comment: https://github.com/ismay/stylelint-no-unsupported-browser-features/issues/245#issuecomment-1140248178

This plugin leverages doiuse for detection, so any updates should be implemented there.

**Comment by ismay** [comment]
More specifically: https://github.com/anandthakker/doiuse/issues/144
